examples/stimuli/tracker_dealer.py

```diff
@@ -52,7 +52,7 @@ def callback(event_type, value=None, timestamp=None):
                    change_criteria, change_rule, x_min, x_max) for i in [0, 1]]
 
 # initialize TrackerDealer object
-tr = TrackerDealer(tr_UD, max_lag, rand)
+tr = TrackerDealer(tr_UD, max_lag, rng)
 
 # Initialize human state
 rng = np.random.RandomState(1)

```

Comment:
Is it stylistically okay that I overwrite rng here? I could see it being confusing to someone just skimming the example.